phase1/aws-ec2-terraform/phase2_implementations/simple-weave-single-master.yaml

```diff
@@ -0,0 +1,271 @@
+#cloud-config
```

Comment:
what is phase2_implementations/? is this basically the handoff between phase1 and phase2 or is this meant to be a full phase2?